nyat64/src/config/arp.rs

```diff
@@ -144,4 +160,53 @@ impl ArpCache {
 
 		Ok(())
 	}
+
+	async fn reply_arp(
+		&self,
+		arp: ArpPacket<'_>,
+		if_mac: MacAddr,
+		mut dst_write: RawPacketStream,
+	) -> Result<()> {
+		let who = arp.get_target_proto_addr();
```

Comment:
This replies to all ARP requests, disregarding which target proto address was queried. Is this intended?